File: frontend/src/@types/ReviewResponse.ts
import { Content } from '@/objects/Content';

export type ReviewResponse = {
  review_id: number;
  created_at: string;
  title: string;
  schedule: ScheduleSimpleResponse;
  contents: Content[];
};

export type Review = {
  review_id: number;
  created_at: string; //ex) "2024-04-22T12:44:00"
  title: string;
  thumbnail_content: Content;
};

File: backend/src/main/java/com/butter/wypl/review/data/response/ReviewResponse.java
package com.butter.wypl.review.data.response;

import java.time.LocalDateTime;
import java.util.Map;

import com.fasterxml.jackson.annotation.JsonProperty;

import lombok.Builder;

@Builder
public record ReviewResponse(

	@JsonProperty("review_id")
	int reviewId,

	@JsonProperty("created_at")
	LocalDateTime createdAt,

	String title,

	@JsonProperty("thumbnail_content")
	Map<String, Object> thumbnailContent
) {

}
